8.2.33 Release # DO NOT DELETE

su-sws-jira opened the pull request.

su-sws-jira committed:
8.2.33

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr833-ikw28zvjxk8wrjeseuik0kwsjqgnmeqw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60523677fcb22d371a73f0a5

Visual Diffs:
- https://dashboard.tugboat.qa/60523677fcb22d371a73f0a5/visualdiffs

Commit:
feat(STN-702): Fix menu border for duplicate dropdown menu links

pookmish committed:
HSD8-1014 Group blocks with attributes and refactor improvements (#834)

* HSD8-1014 Add class attributes to group blocks

* provided update hook for the changes to the group blocks

* cc fixes

Commit:
Feat(STN-690): Bug fix for raised vertical card width (#836)

* feat(STN-690): bug fix for raised vertical card width

Applying width to the div contained within the cards using raised card uniform height, this gives width to the card div wrapper, previously preventing cards with inline elements from going 100%.

* chore: compile css

kaseybon committed:
STN-693: Fix Console Errors (#827)

* fix: resolve js console error

- hb-stretch-vertical-linked-cards JS was running on pages that did not have Vertical Linked Cards

STN-693

* refactor: equal-height-grid.js

- equalHeightGrid() only accepts a list of elements to run against
- Logic for when equalHeightGrid() runs is separated into the index.js file
- Bug fix: equalHeightGrid() will run without errors if a vertical linked card does not have a title
- Set a max-height on cards instead of height to prevent inaccessible text on page resize.

STN-693

* fix: hb-stretch-vertical-linked-card class works on page resize

- stretch class removes previously set min-height
- stretch class is not applied to to small screen sizes
- stretch class min-height is recalculated when the window is resized

STN-693

* chore: compiled JS

su-sws-jira committed:
Updated dependencies Mar 24 2021

pookmish committed:
Sparkbox Sprint 36

pookmish committed:
HSD8-1022 Fixed error with breaking group block form

pookmish committed:
HSD8-1018 Changed the display mode for images on vertical postcards

su-sws-jira committed:
Updated dependencies Mar 31 2021

su-sws-jira committed:
Updated dependencies Apr 7 2021

su-sws-jira committed:
Updated dependencies Apr 14 2021

su-sws-jira committed:
Updated dependencies Apr 21 2021
